jenkins/gce-custom-images/prepare_custom_leonardo_gce_image.sh

```diff
@@ -28,14 +28,16 @@ terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.13"
 terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.0.14"
 terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.0.17"
 
-# leonardo_jupyter will be discontinued soon
 welder_server="us.gcr.io/broad-dsp-gcr-public/welder-server:4d380f2"
 openidc_proxy="broadinstitute/openidc-proxy:2.3.1_2"
 anvil_rstudio_bioconductor="us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:0.0.8"
 
+# Not replaced by Jenkins. If you change this you must also change Leo reference.conf!
+cryptomining_detector="us.gcr.io/broad-dsp-gcr-public/cryptomining-detector:0.0.1"
```

Comment:
shall we add a comment in reference.conf as well saying if you change that, you need to change this as well

Reply:
Yeah, good idea

Reply:
I think we should somehow consolidate all these hashes into fewer places if possible...there're too many places to update right now

Reply:
Yeah +1 -- there's `reference.conf`, automation test `reference.conf`, and the `prepare-*` scripts currently. Also CI/CD updates some of them (e.g. all the terra-docker images) but not others (e.g. rstudio, cryptomining, etc).